src/Magento/FunctionalTestingFramework/Allure/Adapter/MagentoAllureAdapter.php

```diff
@@ -117,4 +118,18 @@ public function stepBefore(StepEvent $stepEvent)
         $this->emptyStep = false;
         $this->getLifecycle()->fire(new StepStartedEvent($stepName));
     }
+
+    /**
+     * Override of parent method, fires StepFailedEvent if step has failed (for xml output)
+     * @param StepEvent $stepEvent
+     * @throws \Yandex\Allure\Adapter\AllureException
+     * @return void
+     */
+    public function stepAfter(StepEvent $stepEvent)
```

Comment:
Getting a warning for this override. Please consider to fix in Allure Adapter code.

Warning: Declaration of Magento\FunctionalTestingFramework\Allure\Adapter\MagentoAllureAdapter::stepAfter(Codeception\Event\StepEvent $stepEvent) should be compatible with Yandex\Allure\Adapter\AllureAdapter::stepAfter() in /Users/jilu/Code/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Allure/Adapter/MagentoAllureAdapter.php on line 24
PHP Warning:  Declaration of Magento\FunctionalTestingFramework\Allure\Adapter\MagentoAllureAdapter::stepAfter(Codeception\Event\StepEvent $stepEvent) should be compatible with Yandex\Allure\Adapter\AllureAdapter::stepAfter() in /Users/jilu/Code/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Allure/Adapter/MagentoAllureAdapter.php on line 24